docs/quickstart.rst

```diff
@@ -13,6 +13,17 @@ A step by step example
 
 .. include:: quickstart_example.rst.inc
 
+Archives and repositories
+-------------------------
+
```

Comment:
Guess we need to explain chunk first:

Chunk: a data (or metadata) block. To implement variable block size deduplication, borg cuts data (and metadata) streams into pieces. These pieces are stored into the repository and are addressed by their content hash.